src/PointwiseFunctions/GeneralRelativity/ProjectionOperators.hpp

```diff
@@ -22,8 +22,8 @@ namespace gr {
  * \ingroup GeneralRelativityGroup
  * \brief Compute projection operator onto an interface
  *
- * \details Returns the operator \f$P^{ij} = g^{ij} - n^i n^j\f$,
- * where \f$g^{ij}\f$ is the inverse spatial metric, and
+ * \details Returns the operator \f$P^{ij} = \gamma^{ij} - n^i n^j\f$,
+ * where \f$\gamma^{ij}\f$ is the inverse spatial metric, and
```

Comment:
isn't that sentence on the line under this comment: "\f$n^i\f$ is the normal vector to the interface in question." Could you clarify what more you want specified?

Reply:
@nilsleiffischer I didn't address this in my fixup since I had the same thought as @nilsdeppe . If there is something specific that you still feel is missing, let me know, and I can add it!

Reply:
Yea that's probably enough